### PR TITLE
feat: Replace the existing Resources search with Azure AI Search 

### PR DESCRIPTION
**Pull request by killij**

### CODE REVIEW

**Review by MartinBelton-gov — APPROVED** [review]

File: Childrens-Social-Care-CPD/Views/Resources/Resource.cshtml [hunk 1]
```diff
@@ -10,6 +10,15 @@
     Layout = "_DefaultPageLayout";
 }
 
+@* @{
+    const int MaxFormatTags = 2;
+
+    var tagInfos = Model.TagMap.Where(x => Model.SearchResult.Document.Tags.Contains(x.Key));
+    var from = tagInfos.Where(x => x.Value.Category == "Resource provider").OrderBy(x => x.Value.DisplayName).FirstOrDefault();
+    var topic = tagInfos.Where(x => x.Value.Category == "Topic").OrderBy(x => x.Value.DisplayName).FirstOrDefault();
```
Comment (rgparkins) [inline Childrens-Social-Care-CPD/Views/Resources/Resource.cshtml:18]: Could this be removed?

File: README.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ The following environment variables are to be configured
 | CPD_AZURE_ENVIRONMENT | The environment for the system |
 | CPD_INSTRUMENTATION_CONNECTIONSTRING | Application insights connection string |
 | CPD_FEATURE_POLLING_INTERVAL | Polling interval for feature configuration udpates |
+| CPD_SEARCH_API_KEY | API Key for the search service |
+| CPD_SEARCH_ENDPOINT | Search service endpoint |
```
Comment (rgparkins) [inline README.md:34]: nice!

**Review by rgparkins — CHANGES_REQUESTED** [review]
Just one comment around commented code being removed

**Review by rgparkins — APPROVED** [review]

**Pull request merged**